src/helpers/__tests__/ci-env/github.json

```diff
@@ -8,14 +8,16 @@
       "GITHUB_RUN_NUMBER": "ghactions-pipeline-number",
       "GITHUB_SHA": "ghactions-commit",
       "GITHUB_WORKFLOW": "ghactions-pipeline-name",
-      "GITHUB_WORKSPACE": "/foo/bar"
+      "GITHUB_WORKSPACE": "/foo/bar",
+      "GITHUB_SERVER_URL": "https://github.com",
+      "GITHUB_RUN_ATTEMPT": "10"
     },
     {
-      "ci.job.url": "https://github.com/ghactions-repo/commit/ghactions-commit/checks",
+      "ci.job.url": "https://github.com/ghactions-repo/actions/runs/ghactions-pipeline-id/attempts/10",
```

Comment:
is this aligned with the `spec_tests`?

Reply:
In this case the code just makes the pipeline URL and the Job URL match wasn't sure if I should change it or not. Should I put the job url back to wha tit was so it matches the spec?

Reply:
I'd say whatever we decide, we should be consistent between pipelines and tests. wdyt?

Reply:
Makes sense I updated this as well as the PR to the spec so that they match.  We still use the check run url but it will always use `GITHUB_SERVER_URL` so that it works on on-prem GitHub enterprise instances